### ISSUE TITLE
Add Settings icon to bottom bar + enable to change settings from prayer

### ISSUE THREAD

**Issue by breviar-sk** [gyim/breviar-ios#37]
Enable to change settings also from opened prayer. Add "settings" icon somewhere to bottom bar (where A+ a- are).

The only setting which must not be changed (so in this case should be hidden or disabled) is Liturgical calendar). All others settings can be changed. 

Optimization: Re-generate only current prayer; re-generate all prayers after returning to main screen.

**Comment by breviar-sk** [comment]
Functionality will be similar to that already implemented: change night/day mode (I hope it is implemented by changing option value?).

**Comment by kominak** [comment]
I'm not sure if bottom toolbar isn't crowded enough - even on iPhone 6. I would put the settings icon to the top right corner.

I wonder what would be better: to present the screen modally (from the bottom) or by pushing (the same way we push the prayer screen from the list of prayers). I think presenting modally is a better approach here.

**Comment by gyim** [comment]
It is very useful to have shortcuts on the toolbar, but it is clear that we cannot put an icon to every single corner of the screen :)

How about having an iBooks-like experience?

- Remove the toolbar and the title from the title bar (the title of the prayer is shown at the beginning of the text, so there is no real need to show it twice)
- Show a single gear icon at the top right corner.  When clicked, a small popup is shown with the settings shortcuts (font size, night mode, maybe voice speed), and a "more options" button that shows the whole Settings screen in a modal dialog
- Show a voice icon next to the gear icon. Also, make it possible to hide this button because it is probably used by a minority of the users, and it is annoying when it is pressed incidentally.

So TL;DR I would have only two icons in the title bar (settings and voice), and hide the toolbar. What do you think?

**Comment by breviar-sk** [comment]
I am not designer so feel free to make design the best way you know. I just want to change settings directly from the prayer :)

**Comment by gyim** [comment]
Implemented a similar feature in v2
